tfx/components/transform/executor.py

```diff
@@ -283,6 +283,8 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
           Tensorflow graph suitable for both training and serving;
         - transformed_examples: Materialized transformed examples, which
           includes both 'train' and 'eval' splits.
+        - cache_output_path: Cache output of 'tf.Transform', where cached
```

Comment:
cache_input_path?

Reply:
its an optional input artifact, so wasnt sure whether ot include it. will add